src/components/Book.js

```diff
@@ -12,7 +19,7 @@ function Book(props) {
             <p className="book-title">{i.genre}</p>
             <p className="book-title">{i.title}</p>
             <p className="book-author">{i.author}</p>
-            <button type="button">Remove</button>
+            <button type="button" onClick={() => removeNow(i)}>Remove</button>
```

Comment:
- [ ] **[OPTIONAL]** While implementing this mapping, you are using the entire book object as the `key` `(<div key={i} className="name-wrap">)`. I would recommend you use something unique like the book `id` property to clear this warning below:

  ![unique-id](https://user-images.githubusercontent.com/36542986/178801740-f35d359a-47dc-410a-b408-405e83e277c2.jpg)